From f38dd83bcf1ed6ccdd83144b6c5976f8a714152d Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sat, 22 Jun 2024 16:03:37 -0700
Subject: [PATCH] Docs/process/index: Remove unsorted docs section

'Other material' section on 'process/index' is no more necessary since
we have 'staging/' directory.  Also all documents on the section has
moved to better places.  Remove the section.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/process/index.rst | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/Documentation/process/index.rst b/Documentation/process/index.rst
index 1827e73f6376..6455eba3ef0c 100644
--- a/Documentation/process/index.rst
+++ b/Documentation/process/index.rst
@@ -107,13 +107,6 @@ developers:
    kernel-docs
    deprecated
 
-These are some overall technical guides that have been put here for now for
-lack of a better place.
-
-.. toctree::
-   :maxdepth: 1
-
-
 .. only::  subproject and html
 
    Indices
-- 
2.39.2

